langchain/src/client/langchainplus.ts

```diff
@@ -203,11 +211,24 @@ export class LangChainPlusClient {
   }
 
   public static async create(
-    apiUrl: string,
+    apiUrl: string | undefined = undefined,
```

Comment:
Would it be too late to make an interface change here and in the constructor?

I think a config object makes more sense given things are under flux and it's bit uncertain what's going to be optional and what isn't:

```
  public static async create(config: {apiUrl?: string, apiKey?: string}) {}
```

Reply:
I usually only like args if it's an important required parameter (`tenantId` might qualify in this case) - LMK and I'm happy to do some refactoring

Reply:
This is the perfect time to do that. I'll go over the other methods to make the requested changes as well